src/server.c

```diff
@@ -574,6 +574,54 @@ null_log_writer (GLogLevelFlags   log_level,
   return G_LOG_WRITER_HANDLED;
 }
 
+/* Bind server to available IPv4 and IPv6 local addresses
+ *
+ * The server must be NOT listening on any interface.
+ *
+ * If port is 0, the server will find an unsued port to listen on. The
+ * port will be the same for both addresses.
+ *
+ * Returns TRUE if the server was binded to at least one address. FALSE
```

Comment:
when you say "was binded" does that mean it is no longer bound.  I suggest changing this to "is bound".

Reply:
True, corrected.